README.md

```
@@ -1,2 +1,186 @@
-# ASSU_BE
-ASSU 백엔드 레포지토리 입니다.
+# 🎓 A:SSU (숭실대학교 종합 제휴 플랫폼)
+<img width="1920" height="1080" alt="예비" src="https://github.com/user-attachments/assets/099ac5d9-2208-4aa3-9cbc-1d4925d1db88" />
+
+
+
+
+> **"제휴의 시작부터 끝까지, A:SSU에서 한 번에!"**  
+> 🏆 **제5회 숭실대학교 IT 프로젝트 공모전 수상작**
+
+[📱 Google Play Store](#) | [💻 GitHub Repository](#) | [📄 API 명세서 (Swagger)](#)
```

markdownlint-cli2 (0.23.1)

[warning] No empty links

(MD042, no-empty-links)